src/AcmLabels/AcmLabels.test.tsx

```diff
@@ -1,14 +1,23 @@
-import React from 'react'
 import { render } from '@testing-library/react'
 import { axe } from 'jest-axe'
+import React from 'react'
 import { AcmLabels } from './AcmLabels'
 
 describe('AcmLabels', () => {
-    test('renders', () => {
+    test('renders empty with undefined', () => {
+        const { container } = render(<AcmLabels labels={undefined} />)
+        expect(container).toMatchInlineSnapshot('<div />')
```

Comment:
Where does the `div` come from?  When I look below I see that it returns a Fragment when it is undefined?

Reply:
Empty fragments in react turn into divs... seems strange.

Reply:
Is there a better way to handle empty than a Fragment?